internal/cluster/validator.go

```diff
@@ -400,6 +401,27 @@ func (v *clusterValidator) allHostsAreReadyToInstall(c *clusterPreprocessContext
 	return ValidationFailure, "The cluster has hosts that are not ready to install."
 }
 
+func (v *clusterValidator) platformRequirementsSatisfied(c *clusterPreprocessContext) (ValidationStatus, string) {
+	// If cluster platform type is not OCI ignore that validation
+	if !(c.cluster.Platform != nil && common.PlatformTypeValue(c.cluster.Platform.Type) == models.PlatformTypeOci) {
+		return ValidationSuccess, "Platform requirements satisfied"
+	}
+
+	usages, err := usage.Unmarshal(c.cluster.Cluster.FeatureUsage)
+	if err != nil {
+		v.log.Errorf("platform validation failure, failed to parse feature usages, %s", err.Error())
+		return ValidationFailure, "Failed to parse feature usages"
+	}
+
+	for _, usg := range usages {
+		if usg.Name == usage.CustomManifest {
```

Comment:
did you verify that in case the user added a custom manifest and then removed it - there is no usage.CustomManifest in the list?

Reply:
I actually just fixed that :)
https://github.com/openshift/assisted-service/pull/5363

Reply:
With https://github.com/openshift/assisted-service/pull/5363 it should, I'll re verify it